src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
That's a good point, and now that I've done a bit more reading I'm not sure if the way that R implements these really does what I thought it does (in terms of interpretation of the coefficients). So I'll take out the explicit mention of R now since our implementation might well change in the future. We might want to consider a "getting started" guide that translates things for R, SAS, etc. users, but I'm not sure that this is the best place for that.